addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js

```diff
@@ -89,10 +24,10 @@ registry.category("web_tour.tours").add('sale_timesheet_tour', {
     content: 'Click on "Add a product" to add a new product. We will add a service product.',
 }, {
     trigger: '.o_field_html[name="product_id"], .o_field_widget[name="product_template_id"] input',
-    content: 'Select a prepaid service product (e.g. Service Product Prepaid Hours)',
-    run: 'text Service Product Prepaid Hours',
+    content: Markup('Select a prepaid service product <i>(e.g. Service Product (Prepaid Hours))</i>'),
```

Comment:
Hello @odoo/rd-security ,

Can we have an override for this PR? 

We use the Markup function with a static content in a JS tour used in a python test. 🙂 

Thanks in advance!

Have a nice day! 🙂

Reply:
Coucou @mart-e 👋🏼 

Can we have an security override for this PR?
We use Markup function with a static content in a JS tour used in a python test. 🙂 

Thanks in advance! 

Have a nice afternoon! 🙂